supervisor/errata_utils.py

```diff
@@ -57,6 +57,27 @@ def ET_api_post(path: str, data: dict):
     return response.json()
 
 
+def ET_api_put(path: str, data: dict, send_as_json: bool = True):
```

Comment:
It didn't occur to me that we were posting as form data in other cases, but we are. Let's be consistent. The way you do this as form data, is you post with 

```
 data = {
    "advisory[product_owner_email]": "jotnar-bot@IPA.REDHAT.COM"
    ...
 }
```